pyOCD/board/mbed_board.py

```diff
@@ -83,6 +83,7 @@ def __init__(self, link, target=None, frequency=1000000):
         """
         self.native_target = None
         self.test_binary = None
+        link.set_deferred_transfer(True)
```

Comment:
The Board class still makes calls to pyDAPAccess via the link object. It calls things like set_clock, disconnect and close. Setting deferred shouldn't be too much different than a call to set_clock.